### PR TITLE
 adding delete student of cohort funtion

### PR DESCRIPTION
**Pull request by HemaSAli**
relates #14

### CODE REVIEW

**Review by ali-7 — APPROVED** [review]

**Review by lubnaabd — APPROVED** [review]

File: public/js/cohortStudent.js [hunk 1]
```diff
@@ -62,3 +63,34 @@ addStudentButton.addEventListener('click', (e) => {
     swal('Please Enter Data ! ! ', '', 'error');
   }
 });
+
+deleteStudentButton.forEach((element) => {
```
Comment (NouraldinS) [inline public/js/cohortStudent.js]: @lubnaabd Had made a function like this in `delCohort.js`, see the similarities and put them in one function, then call them back here @HemaSAli .

File: src/database/queries/deleteCohortStudent.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+const dbConnection = require('../db_connection');
+
+const deleteStudentQuery = (deleteStdData, cb) => {
+  const { stdId, cohortID } = deleteStdData;
```
Comment (NouraldinS) [inline src/database/queries/deleteCohortStudent.js]: You could also say:
```js
const deleteStudentQuery = ({ stdId, cohortID }, cb) => {
```
But yours work too, so no need to change it.

**Review by NouraldinS — CHANGES_REQUESTED** [review]

File: public/js/cohortStudent.js (hunk 1 above)
Comment (HemaSAli) [reply public/js/cohortStudent.js]: I saw what lubna made , I think they are the same 
and here is lubna code 
```
Array.prototype.forEach.call(del, (element) => {
  element.addEventListener('click', (event) => {
```

Comment (NouraldinS) [reply public/js/cohortStudent.js]: I know they're the same, don't have repeated code. Pick Lubna's code and assign it to a function, then use it in your code and Lubna's.

File: src/database/queries/deleteCohortStudent.js (hunk 2 above)
Comment (HemaSAli) [reply src/database/queries/deleteCohortStudent.js]: thanks for this idea , I did it

File: public/js/cohortStudent.js [hunk 3]
```diff
@@ -6,11 +6,13 @@ const loading = document.getElementById('loading');
 const URL = window.location.href;
 const splitUrl = URL.split('/');
 const cohortID = splitUrl[splitUrl.length - 2];
```
Comment (NouraldinS) [inline public/js/cohortStudent.js]: Change name to `cohortId`

File: public/js/cohortStudent.js [hunk 4]
```diff
@@ -6,11 +6,13 @@ const loading = document.getElementById('loading');
 const URL = window.location.href;
```
Comment (NouraldinS) [inline public/js/cohortStudent.js]: change name to `Url`

File: public/js/cohortStudent.js [hunk 5]
```diff
@@ -6,11 +6,13 @@ const loading = document.getElementById('loading');
...
 const splitUrl = URL.split('/');
 const cohortID = splitUrl[splitUrl.length - 2];
+const deleteStudentButton = document.querySelectorAll('.delete');
 
 const showAddStudentDiv = () => {
```
Comment (NouraldinS) [inline public/js/cohortStudent.js]: change name to `toggleAddStudentDiv`

**Review by NouraldinS — CHANGES_REQUESTED** [review]

**Review by NouraldinS — APPROVED** [review]

**Pull request merged**